huntsman/utils/sshfs_mount.py

```
@@ -0,0 +1,109 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Dec  3 07:16:00 2019
+
+@author: danjampro
+
+Run this on the pis (not the main computer).
+"""
+import os, subprocess
+from huntsman.utils import load_config
+
+#==============================================================================
+
+def mount(mountpoint, remote, server_alive_interval=20,
+          server_alive_count_max=3, logger=None):
+    '''
+    Mount remote on local.
+    '''
+    if logger is None:
+        logger = print
+        
+    if not os.path.isdir(mountpoint):
+        try:
+            os.makedirs(mountpoint, exist_ok=True)
+        except FileExistsError:
+            pass #For some reason this is necessary
+        
+    options = f'ServerAliveInterval={server_alive_interval},' + \
+              f'ServerAliveCountMax={server_alive_count_max}'
+    options = ['sshfs', remote, mountpoint, '-o', options]
+    try:
+        subprocess.run(options, shell=False, check=True)
+    except Exception as e:
+        logger(f'Failed to mount {remote} on {mountpoint}.')
+        raise(e)
+    
+    
+def unmount(mountpoint):
+    '''
+    Unmount remote from local.
+    '''
+    if os.path.isdir(mountpoint):
+        options = ['fusermount', '-u', mountpoint]
+        try:
+            subprocess.run(options, shell=False, check=True)
+        except:
+            pass
+        
+#==============================================================================
+
+def mount_sshfs(logger=None, user=None):
+    '''
+    
+    '''
+    if logger is None:
+        logger = print
+
+    #Get SSH username from environment?
+    if user is None:
+          user = os.environ['PANUSER']
```

Might want to throw an exception here if user has not been specified and the environment variable hasn't been set either.  A `warnings.warn()` at the very least.